library/src/main/java/com/getstream/sdk/chat/viewmodel/messages/MessageListItemLiveData.java

```diff
@@ -50,19 +57,25 @@ public MessageListItemLiveData(User currentUser,
         this.currentUser = currentUser;
         this.typing = typing;
         this.reads = reads;
-        this.messageEntities = new ArrayList<MessageListItem>();
+        this.messageEntities = new ArrayList<>();
         this.typingEntities = new ArrayList<>();
         this.isLoadingMore = false;
         // scroll behaviour is only triggered for new messages
         this.lastMessageID = "";
         this.hasNewMessages = false;
```

Comment:
After reviewing this class, I can see this field is not needed anymore. We are always to updating it before we call to `broadcastValue()` method. 
There is a public method that returns this value, but it is not used anymore, and I think it doesn't provide any useful value, so we can remove it

Reply:
We need it to broadcast the current state of message entities, like:

```
private void onReadsChanged(List<ChannelUserRead> channelUserRead) {
        hasNewMessages = false;
        logger.logI("broadcast because reads changed");
        broadcastValue(hasNewMessages, isLoadingMore, typingEntities, messageEntities);
    }
```

Unless we can broadcast this as an empty list.

Reply:
Yep, but you only need to pass it on the method, it is not needed to be stored into the class

Reply:
fixed